base/ltoutput.dtx

```diff
@@ -2895,17 +2897,29 @@
   \language\document@default@language
 %    \end{macrocode}
 %
-%    The |\catcode`\ = 10| was removed as it was considered useless
-%    (presumably because nothing gets tokenized during shipout).
-%
-%    This was put in as some error produced active spaces in a mark, I
-%    think.
-%
-%    Why was the hyphen reset?
-%
+%    Now a few more resets for things that may have changed during
+%    galley processing. For some active character definitions.
 %    \begin{macrocode}
   \@resetactivechars
 %    \end{macrocode}
```

Comment:
I cannot parse this! And the second sentence is not a sentence. 

For example: to what does "Now a few resets" refer?

Rewrite, please!